mne/viz/evoked.py

```diff
@@ -645,6 +664,42 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
     fig : instance of matplotlib.figure.Figure
         Images of evoked responses at sensor locations
     """
+    if not type(evoked) in (tuple, list):
+        evoked = [evoked]
+
+    if dark_background is not None:
+        if dark_background:
+            fig_facecolor = 'k'
+            axis_facecolor = 'k'
+            font_color = 'w'
+        else:
+            fig_facecolor = 'w'
+            axis_facecolor = 'w'
+            font_color = 'k'
+
+        if color is None:
+            if dark_background:
+                color = ['w'] + COLORS
+            else:
+                # default colors from M Waskom's Seaborn
+                color = ['#e41a1c', '#377eb8', '#4daf4a', '#984ea3', '#ff7f00',
+                         '#1b9e77', '#d95f02', '#7570b3', '#e7298a', '#66a61e']
+            color = color * ((len(evoked) % len(color)) + 1)
+            color = color[:len(evoked)]
+
+    if fig_facecolor is None:
+        fig_facecolor = 'k'
+        warn('axis_facecolor is deprecated and will be removed in v0.16. Use '
+             'dark_background parameter or change matplotlib defaults.')
+    if axis_facecolor is None:
+        axis_facecolor = 'k'
+        warn('axis_facecolor is deprecated and will be removed in v0.16. Use '
+             'dark_background parameter or change matplotlib defaults.')
+    if font_color is None:
+        font_color = 'w'
+        warn('font_color is deprecated and will be removed in v0.16. Use '
+             'dark_background parameter or change matplotlib defaults.')
```

Comment:
These warnings get thrown when using the defaults. Shouldn't it be the other way round.